.idea/codeStyleSettings.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why you have .idea files in repository?
Everytime I see this unnecessary history